assets/JS03-04.js

```diff
@@ -56,13 +56,20 @@ function createStatusBtn( value, div, index ) {
 
 //出力内容をすべて消去して、再度出力
 function reset() {
-  output.textContent = "";
+  output.textContent = '';
   outputHtml();
+  formClear();
+}
+
+//フォーム入力値を空にする
+function formClear() {
+  let input = document.getElementById('inputComment');

```

Comment:
再代入しないので定数で宣言できます。
ここにだけに限らず今後も可能な限り定数で宣言するように心がけましょう。
```suggestion
  const input = document.getElementById('inputComment');
```